src/main/java/com/smartbear/readyapi/client/TestRecipeBuilder.java

```diff
@@ -54,5 +54,14 @@ public static TestRecipeBuilder newTestRecipe() {
         return new TestRecipeBuilder();
     }
 
+    public static TestRecipeBuilder newTestRecipe(Object... args) {
+        TestRecipeBuilder recipeBuilder = newTestRecipe();
+        for (Object arg : args) {
```

Comment:
don't get me wrong - I'd love to go 1.8 :-) perhaps if we add that "extractor" thing!?